src/test/suite/participant/participant.test.ts

```diff
@@ -1321,6 +1321,49 @@ suite('Participant Controller Test Suite', function () {
               sinon.restore();
             });
 
+            test('shows an error if something goes wrong with getting databases', async function () {
```

Comment:
there is a lot of copy-pasting here with these tests and little change other than the command string, I could turn this into a test matrix but it'd break the structure pattern of tests.